kadi/commands/tests/test_commands.py

```diff
@@ -372,7 +372,8 @@ def stop_date_fixture(monkeypatch):
     return stop_date_fixture
 
 
-stop_date_2021_10_24 = stop_date_fixture_factory("2021-10-24")
+# 2021:297 0300z just after recovery maneuver following 2021:296 NSM
+stop_date_2021_10_24 = stop_date_fixture_factory("2021-10-24 03:00:00")
```

Comment:
Though I'm guessing this is constructed to hit the OBS bug?

Reply:
This is unrelated to the OBS bug.

The original test was passing only because of the issue where all command events (including those after the stop date) were included. This change includes additional event(s) that are required for the original test to pass.